Sources/XcbeautifyLib/Renderers/OutputRendering.swift

```diff
@@ -565,4 +565,80 @@ extension OutputRendering {
+        return "Test run with \(group.numberOfTests) tests passed after \(group.totalTime) seconds"
+    }
+
+    func formatSwiftTestingRunFailed(group: SwiftTestingRunFailedCaptureGroup) -> String {
+        return "Test run with \(group.numberOfTests) tests failed after \(group.totalTime) seconds with \(group.numberOfIssues) issue(s)"
+    }
+
+    func formatSwiftTestingSuiteStarted(group: SwiftTestingSuiteStartedCaptureGroup) -> String {
+        return "Suite \(group.suiteName) started"
+    }
+
+    func formatSwiftTestingTestStarted(group: SwiftTestingTestStartedCaptureGroup) -> String? {
+        return nil
+    }
+
+    func formatSwiftTestingSuitePassed(group: SwiftTestingSuitePassedCaptureGroup) -> String {
+        return "Suite \(group.suiteName) passed after \(group.timeTaken) seconds"
+    }
+
+    func formatSwiftTestingSuiteFailed(group: SwiftTestingSuiteFailedCaptureGroup) -> String {
+        return "Suite \(group.suiteName) failed after \(group.timeTaken) seconds with \(group.numberOfIssues) issue(s)"
+    }
+
+    func formatSwiftTestingTestFailed(group: SwiftTestingTestFailedCaptureGroup) -> String {
+        let message = "\(group.testName) (\(group.timeTaken) seconds) \(group.numberOfIssues) issue(s)"
+        let wholeMessage = colored ? TestStatus.fail.foreground.Red + " " + message.f.Red : TestStatus.fail + " " + message
+        return Format.indent + wholeMessage
+    }
+
+    func formatSwiftTestingTestPassed(group: SwiftTestingTestPassedCaptureGroup) -> String {
+        let testName = group.testName
+        let timeTaken = group.timeTaken
+        return colored ? Format.indent + TestStatus.pass.foreground.Green + " " + testName + " (\(timeTaken.coloredTime()) seconds)" : Format.indent + TestStatus.pass + " " + testName + " (\(timeTaken) seconds)"
+    }
+
+
+    func formatSwiftTestingTestSkipped(group: SwiftTestingTestSkippedCaptureGroup) -> String {
+        let testName = group.testName + " skipped"
+        return colored ? Format.indent + TestStatus.skipped.foreground.Yellow + " " + testName : Format.indent + TestStatus.skipped + " " + testName
+    }
+
+    func formatSwiftTestingTestSkippedReason(group: SwiftTestingTestSkippedReasonCaptureGroup) -> String {
+        let testName = group.testName + " skipped"
+        let reason = group.reason.map { " (\(String($0)))" } ?? ""
+        return colored ? Format.indent + TestStatus.skipped.foreground.Yellow + " " + testName + reason : Format.indent + TestStatus.skipped + " " + testName + reason
+    }
+
+    func formatSwiftTestingIssue(group: SwiftTestingIssueCaptureGroup) -> String {
+        let issueDetails = group.issueDetails != nil ? " at \(group.issueDetails!)" : ""
+        let message = " Test [\(group.testDescription)] recorded an issue\(issueDetails)"
+        return colored ? Format.indent + Symbol.warning.f.Yellow + " " + message : Format.indent + Symbol.asciiWarning + " " + message
+    }
```

Comment:
Can you add an implementation for the GitHub Actions and TeamCity renderers? I think those one should annotate a warning or error for each.